tag/views.py

```
@@ -0,0 +1,58 @@
+ 
+ 
+class TagDetailView(APIView):  
+    @swagger_auto_schema(
+      operation_id='태그 내부 게시물 조회',
+      operation_description='해당 태그가 달린 게시물을 조회합니다.',
+      responses={200: PostSerializer(many=True), 204: 'No Content'}
+    )
+    def get(self, request, tag_id):
+      try:
+        Tag.objects.get(id=tag_id)
+      except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)